### PR TITLE
Fix link to Sulu CMS

### PR DESCRIPTION
**Pull request by alexander-schranz**
The correct repository to track Sulu commits is [`sulu/sulu`](https://github.com/sulu/sulu) repository.

- [x] verified that the CMS I'm adding is still maintained.
- [x] read [CONTRIBUTING.md](https://github.com/postlight/awesome-cms/blob/master/CONTRIBUTING.md).
- [x] did not generate README.md.

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -667,7 +667,6 @@
       "avatar_url": "https://avatars3.githubusercontent.com/u/1058963?v=4",
       "profile": "https://thingsthat.com",
       "contributions": [
-        "docs",
```
Comment (alexander-schranz) [inline .all-contributorsrc:670]: This fails with unknown type in the generation process.

Comment (skoch) [reply .all-contributorsrc:670]: Nice catch, thank you!

**Comment by alexander-schranz** [comment]
@skoch thx, do I need todo aomething to trigger the rebuild of the readme?

**Comment by alexander-schranz** [comment]
@skoch looks like there is an error when it tries to push the changes in the CI:

```
remote: error: GH006: Protected branch update failed for refs/heads/main.        
remote: 
remote: - Changes must be made through a pull request.        
To https://github.com/postlight/awesome-cms
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/postlight/awesome-cms'
Error: Invalid status code: 1
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/index.js:17:19)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5) {
  code: 1
}
Error: Invalid status code: 1
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4/index.js:17:19)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5)
```

**Comment by skoch** [comment]
Yeah, I saw that yesterday as well. I believe the issue is due to the fact that `main` is protected. 

https://github.com/stefanzweifel/git-auto-commit-action?tab=readme-ov-file#push-to-protected-branches


I have the PAT added as a secret ... now I have to find the time to update the workflow and test.

**Pull request merged**